app/src/main/java/com/pajato/android/gamechat/common/ToolbarManager.java

```diff
@@ -202,6 +199,18 @@ private void addMenuItem(@NonNull final Toolbar toolbar, @NonNull final MenuItem
         item.setShowAsAction(type.flag);
     }
 
+    /** Reset the current toolbar overflow menu */
+    public void resetOverflowMenu(@NonNull Resources resources, final ToolbarType type,
```

Comment:
I think you can safely shorten this name to "reset" without losing clarity since it is the only reset that takes place in the toolbar manager class (that I've seen so far).